packages/block-library/src/media-text/edit.js

```diff
@@ -122,228 +103,195 @@ class MediaTextEdit extends Component {
 			href: newHref,
 			focalPoint: undefined,
 		} );
-	}
+	};
+}
 
-	onWidthChange( width ) {
-		this.setState( {
-			mediaWidth: applyWidthConstraints( width ),
-		} );
-	}
+function MediaTextEdit( { attributes, className, isSelected, setAttributes } ) {
+	const {
+		focalPoint,
+		href,
+		imageFill,
+		isStackedOnMobile,
+		linkClass,
+		linkDestination,
+		linkTarget,
+		mediaAlt,
+		mediaId,
+		mediaPosition,
+		mediaType,
+		mediaUrl,
+		mediaWidth,
+		rel,
+		verticalAlignment,
+	} = attributes;
+
+	const image = useSelect(
```

Comment:
Shouldn't this be { image } if you return an object below?

Reply:
True, I must have forgotten to test it. I now confirmed that the `<ImageURLInputUI>` component which used the object works.